pages/ksphere/kommander/1.2/ldap/index.md

```diff
@@ -1,11 +1,11 @@
 ---
 layout: layout.pug
-beta: true
+beta: false
 navigationTitle: LDAP Directory
 title: External LDAP Identity Provider
 menuWeight: 10
 excerpt: Use your LDAP Identity Provider with Konvoy and Kommander
-beta: true
+beta: false
```

Comment:
@gracedo or @cmastr 
Let's take out this second one - it kind of gets wacky if it has two tags like this.

```suggestion
```